<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Squick Helper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,1">
    <item>
     <widget class="QGroupBox" name="gb_opencom">
      <property name="title">
       <string>Open device</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,0,1,0,0">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Com port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_com_port"/>
       </item>
       <item>
        <widget class="QPushButton" name="b_com_update">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Baudrate:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_baudrate">
         <property name="currentText">
          <string>9600</string>
         </property>
         <property name="currentIndex">
          <number>3</number>
         </property>
         <item>
          <property name="text">
           <string>1200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_com_connect">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_com_disconnect">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Disconnect</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="chart_layout"/>
    </item>
    <item>
     <widget class="QGroupBox" name="gb_log">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Log</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QProgressBar" name="pb_progress">
           <property name="value">
            <number>0</number>
           </property>
           <property name="textVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextBrowser" name="t_cmd"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="le_cmd"/>
         </item>
         <item>
          <widget class="QPushButton" name="b_send_cmd">
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>522</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionDownload_geiger_data"/>
    <addaction name="actionOpen_monitor_mode"/>
    <addaction name="actionUpdate_firmware"/>
   </widget>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionDownload_geiger_data">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Download geiger data</string>
   </property>
  </action>
  <action name="actionOpen_monitor_mode">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open monitor mode</string>
   </property>
  </action>
  <action name="actionUpdate_firmware">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Update firmware</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
